src/utils/convert.js

```diff
@@ -87,6 +87,10 @@ const getAuthType = (definition) => {
   return authTypeMap[definition.general.auth_type];
 };
 
+const hasAuth = (definition) => {
```

Comment:
Could this be moved to the `getAuthMetaData` as a returned property?

Reply:
They are separate functions because in some situations we only need to quickly check if the app needs authentication or not. I don't want to do all the unnecessary things in `getAuthMetaData` for those situations. Makes sense?

Reply:
Alright!